content/source/docs/plugin/framework/schemas.html.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: "extend"
+page_title: "Plugin Development - Framework: Schemas"
+description: |-
+  How to define a schema using the next-generation provider development
+  framework.
+---
+
+# Schemas
+
+Terraform providers can specify the constraints of Terraform configuration
+blocks through the use schemas. These schemas define what fields a provider,
+resource, or data source configuration block has, and give Terraform metadata
+about those fields. The schema can be thought of as the "type information" or
+the "shape" of a resource, data source, or provider.
+
+In the next-generation framework, schemas are defined using [the
+`schema.Schema`
+struct](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/schema#Schema)
+and returned from methods on the
+[provider](/docs/plugin/framework/providers.html),
+[resources](/docs/plugin/framework/resources.html) or [data
+sources](/docs/plugin/framework/data-sources.html).
+
+## Version
+
+Every schema has a version. The version is an integer that allows provider
+developers to track changes to their schemas. It is generally only used when
```

Comment:
We should address the user as "you" here. We can assume that folks reading this are provider developers. 

```suggestion
Every schema has a version, which is an integer that allows you to track changes to your schemas. It is generally only used when
```